sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -170,24 +169,23 @@ FormRecognizerAudience getAudience() {
      *         &#125;&#41;;
      *     &#125;&#41;;
      * </pre>
-     * <!-- end com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.beginBuildModel#String-String -->
+     * <!-- end com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.beginBuildModel#String-DocumentBuildMode -->
      *
-     * @param trainingFilesUrl source URL parameter that is an externally accessible Azure
-     * storage blob container Url (preferably a Shared Access Signature Url).
+     * @param trainingFilesUrl an Azure Storage blob container's SAS URI. A container URI (without SAS)
+     * can be used if the container is public or has a managed identity configured. For more information on
+     * setting up a training data set, see: <a href="https://aka.ms/azsdk/formrecognizer/buildtrainingset">here</a>.
```

Comment:
@kinelski created a new link for this, for v3 specifically, https://aka.ms/azsdk/formrecognizer/buildcustommodel